livekit_analytics.proto

```diff
@@ -38,6 +51,8 @@ message AnalyticsStat {
   float connection_score = 16;
   string track_id = 17;
   uint32 rtt = 18;
+  uint64 total_frames = 19;
```

Comment:
I was thinking of this as aggregate of all layers. If we wanted to show the fps of the highest layer only, it can be determined from the `streams` array below. As layers could come and go, sampling every 5 seconds could get tricky. For example, highest layer could have started at 4 seconds into the measurement window and done 30 fps. But, averaging over the 5 second sampling window, fps could look really low. How should we accommodate for that?

Reply:
we don't need this field. we can accumulate from streams downstream. since every layers data is available in an interval we should be covered to either get total and/or specific layer

Reply:
Thank you @shishirng . I know it will makes things incompatible and a breaking change. But, given that this is not released yet, wondering if we can clean up all the layer specific fields in this message and just move to an array of `AnalyticsStream` here? Like you said, down stream processing can do the necessary aggregation/filtering by taking data from layers.

Reply:
@boks1971 that should be fine (breaking changes). For audio, im assuming 1 of the layers will be used in the stream?

Reply:
That's great @shishirng . Let me clean it up and push up a change for your review.

If we are going the streams route. I will just use it for everything. For non-simulcast, there will be just be one stream with layer set to 0 (i. e. audio streams and video streams which do not use simulcast like screen share).

Reply:
Made this commit to refactor - https://github.com/livekit/protocol/pull/86/commits/d16770b5fa695b460cb8574a2104429df3af51ed

Also added explanation in this comment - https://github.com/livekit/protocol/pull/86/commits/d16770b5fa695b460cb8574a2104429df3af51ed#r800082733

Let me know.